package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "opentok-network-test-js",
+  "name": "vonage-network-test-js",
```

Comment:
This should be the @vonage`-scoped package name (here, in package-lock.json, and in the sample app). Please find out what we want the name to be. From the name of the repo, I think `@vonage/video-api-network-test` would work.